opensrp-chw/src/main/java/org/smartregister/chw/reporting/ChwReport.java

```diff
@@ -139,23 +140,25 @@ public static void createAncReportViews(ViewGroup mainLayout, List<Map<String, I
 
     public static void createChildReportViews(ViewGroup mainLayout, List<Map<String, IndicatorTally>> indicatorTallies) {
         //Disclaimer: Pie charts have binary slices yes and no with different tallying done separately ;)
+        if (!BuildConfig.FLAVOR.equals("lmh")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5698309)